db/transaction/mocks/Transaction.go

```diff
@@ -107,6 +107,29 @@ func (_m *Transaction) Fetch(_a0 *schema.Schema, _a1 transaction.Filter) (*schem
 	return r0, r1
 }
 
+// LockFetch mock
+func (_m *Transaction) LockFetch(_a0 *schema.Schema, _a1 transaction.Filter) (*schema.Resource, error) {
```

Comment:
sort of auto generated (have a look at whole file). BTW @mmatczuk would probably not agree about the naming ;)